website/components/basic-hero/style.css

```diff
@@ -42,6 +50,22 @@
        * The left and right excess is left as is - it's needed for proper centering when wrapping.
        */
       margin: 8px;
+
+      /*
+       * Overrides for tertiary links. Otherwise they would use brand colors
+       * and be too close to the other links.
+       */
+      &.variant-tertiary {
+        margin-top: 16px;
+
+        &:not(.background-brand):not([disabled]).brand-consul {
+          color: var(--gray-4);
+
+          & .icon svg [stroke] {
+            stroke: var(--gray-4);
+          }
+        }
+      }
```

Comment:
So this CSS override here could actually be completely avoided if you took advantage of slices in JSX.

Slices are really great for scenarios like this, as they gracefully degrade when the array isn't log enough, as they just return `[]` (which will work as desired when mapped over).

```jsx
<div className="links">
  {links.slice(0, 2).map((link, stableIdx) => {
    {/* Old link logic goes here */}
  })
</div>

{/* I'm not sure yet if this should be inside of outside of links, but outside coud simplify the above css max width*/}
{links.slice(2, 3).map((link, stableIdx) => {
  {/* New link logic goes here, and you can appropriately theme this link consistently */}
})
```